kinescope/downloader.py

```diff
@@ -94,15 +94,23 @@ def _get_license_key(self) -> str:
     def _fetch_segment(self,
                        segment_url: str,
                        file):
-        for _ in range(5):
+        print("segment_url: {segment_url}, file: {file}".format(segment_url = segment_url, file = file))
```

Comment:
Foramat is not recommended for use. I would suggest you to use f-string or redo the logging system at the root :)